adapter_v1_20_R3/src/main/java/com/playmonumenta/redissync/adapters/VersionAdapter_v1_20_R3.java

```diff
@@ -0,0 +1,302 @@
+package com.playmonumenta.redissync.adapters;
+
+import ca.spottedleaf.dataconverter.minecraft.MCDataConverter;
+import ca.spottedleaf.dataconverter.minecraft.datatypes.MCTypeRegistry;
+import com.google.gson.Gson;
+import com.google.gson.JsonArray;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.playmonumenta.mixinapi.v1.RedisSyncIO;
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.util.Map;
+import java.util.logging.Logger;
+import javax.annotation.Nullable;
+import net.minecraft.SharedConstants;
+import net.minecraft.nbt.*;
+import net.minecraft.world.scores.Scoreboard;
+import org.bukkit.craftbukkit.v1_20_R3.scoreboard.CraftScoreboard;
+import org.bukkit.entity.Player;
+
+public class VersionAdapter_v1_20_R3 implements VersionAdapter {
+	@Nullable
+	private static Gson advancementsGson = null;
+
+	/**
+	 * Creates the version adapter.
+	 *
+	 * @param logger The logger to use
+	 */
+	public VersionAdapter_v1_20_R3(Logger logger) {
```

Comment:
I need to suppress this, but can't figure out how... we never use it, and it only exists because of reflection. I wonder if `invoke` discards extra arguments.